core/src/main/java/org/elasticsearch/common/blobstore/BlobContainer.java

```diff
@@ -37,14 +36,19 @@
     boolean blobExists(String blobName);
 
     /**
-     * Creates a new {@link InputStream} for the given blob name
+     * Creates a new OutputStream for the given blob name
```

Comment:
InputStream or OutputStream? This is confusing because the method is `InputStream readBlob(String blobName)`